v2/common/pom.xml

```diff
@@ -119,6 +119,10 @@
           <groupId>com.google.cloud</groupId>
           <artifactId>google-cloud-kms</artifactId>
         </dependency>
+        <dependency>
+          <groupId>com.google.protobuf</groupId>
+          <artifactId>protobuf-java</artifactId>
+        </dependency>
```

Comment:
Is this necessary? No new imports were added to the corresponding files

Reply:
I was following https://www.xolstice.org/protobuf-maven-plugin/usage.html explicitly asked to add these lines.

It wasn't introducing issue because protobuf-java has always been a transitive dependency of other declared dependencies, even though v2/commons also directly used protobuf without declare it.

Hoever in general one should declare a dependency if it is directly used in that module